src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Monolog\Handler;
+
+use RuntimeException;
+use Monolog\Logger;
+
+/**
+ * Handler send logs to Telegram using Telegram Bot API.
+ *
+ * How to use:
+ *  1) Create telegram bot with https://telegram.me/BotFather
+ *  2) Create a telegram channel where logs will be recorded.
+ *  3) Add created bot from step 1 to the created channel from step 2.
+ *
+ * Use telegram bot API key from step 1 and channel name with '@' prefix from step 2 to create instance of TelegramBotHandler
+ *
+ * @link https://core.telegram.org/bots/api
+ *
+ * @author Mazur Alexandr <alexandrmazur96@gmail.com>
+ */
+class TelegramBotHandler extends AbstractProcessingHandler
+{
+    private const BOT_API = 'https://api.telegram.org/bot';
+
+    /**
+     * Telegram bot access token provided by BotFather.
+     * Create telegram bot with https://telegram.me/BotFather and use access token from it.
+     * @var string
+     */
+    private $apiKey;
+
+    /**
+     * Telegram channel name.
+     * Since to start with '@' symbol as prefix.
+     * @var string
+     */
+    private $channel;
+
+    /**
+     * @param string $apiKey Telegram bot access token provided by BotFather
+     * @param string $channel Telegram channel name
+     * @inheritDoc
+     */
+    public function __construct(
+        string $apiKey,
+        string $channel,
+        $level = Logger::DEBUG,
+        bool $bubble = true
+    ) {
+        parent::__construct($level, $bubble);
+
+        $this->apiKey = $apiKey;
+        $this->channel = $channel;
+        $this->level = $level;
+        $this->bubble = $bubble;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function write(array $record): void
+    {
+        $this->send($record['formatted']);
+    }
+
+    /**
+     * Send request to @link https://api.telegram.org/bot on SendMessage action.
+     * @param string $message
+     */
+    protected function send(string $message): void
+    {
+        $ch = curl_init();
+        $url = self::BOT_API . $this->apiKey . '/SendMessage';
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query([
+            'text' => $message,
+            'chat_id' => $this->channel,
+        ]));
+
+        $result = curl_exec($ch);
```

Comment:
Please use `Curl\Util`, see MandrillHandler or SlackWebhookHandler for example usages.

Reply:
Thank you for your reply! 
I looked at Curl\Util and find that `execute()` does not return any response from the server and only throws an exception when got curl error. 

But, Telegram API send error in the response body and I think would be great if we say to the user what exactly wrong.

Maybe would be better add a method to the `Curl\Util` same as `execute` but with response return from `curl_exec()` call?

Reply:
I think we could change Curl\Util::execute to always return the result, as it throws otherwise. The return type would then be `string` or `true` depending on CURLOPT_RETURNTRANSFER.

Reply:
I create a [pull request](https://github.com/Seldaek/monolog/pull/1341) for this feature.

I can fix this after merging it, or I can merge this pull request into this branch (the bad way I guess).

Reply:
Merged it, please rebase this :)

Reply:
Rebased and apply changes.
I retest my code - all good ;)